packages/amplify-category-notifications/src/display-utils.ts

```diff
@@ -0,0 +1,38 @@
+import { spinner } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+
+/**
+ * Display warning all channels have been enabled
+ */
+export const viewShowAllChannelsEnabledWarning = (): void => {
+  printer.info('All the available notification channels have already been enabled.');
+};
+/**
+ * Display warning that amplify push is required to enable the channel
+ */
+export const viewShowDeferredModeInstructions = (): void => {
+  printer.warn('Run "amplify push" to update the channel in the cloud');
+};
+
+/**
+ * Display status that Auth and Pinpoint resources are being deployed to the cloud
+ */
+export const viewShowInlineModeInstructionsStart = async (channelName: string): Promise<void> => {
+  spinner.start(`Channel ${channelName} requires a Pinpoint resource in the cloud. Proceeding to deploy Auth and Pinpoint resources...`);
```

Comment:
do we need to do anything to ensure this spinner gets stopped eventually? (even if an error is thrown)

Reply:
the usage of this method is always used in a try catch together with `viewShowInlineModeInstructionsFail`. In order to enforce it maybe we can ensure the spinner failed/closes in our exception handler?